src/ec/packages/ec-component-page-header/page-header.html.twig

```diff
@@ -0,0 +1,69 @@
+{% spaceless %}
+{#
+  Parameters:
+    - "title" (string) (default: '') Title of header
+    - "description" (string) (default: '') Description of header
+    - "meta" (string) (default: '') Meta of header
+    - "infos" (array) (default: []) Array of infos. format: [
+        {
+            "text" (string) Label of info
+            "icon" (associative array) Predefined structure compatible with EC Icon
+        },
+        ...
+        ]
+    - "breadcrumb" (associative array) (default: '') Predefined structure compatible with EC Breadcrumb
+    - "extra_classes" (string) (default: '')
+    - "extra_attributes" (array) (default: []): format: [
+        {
+          "name" (string) (default: ''),
+          "value" (string) (default: '')
+        },
+        ...
+      ]
+#}
+
+{# variables #}
+{% set _title = title|default('') %}
+{% set _meta = meta|default('') %}
+{% set _description = description|default('') %}
+
+{% set _css_class = 'ecl-page-header' %}
+{% set _extra_attributes = '' %}
+
+{# Internal logic - Process properties #}
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
+  {% for attr in extra_attributes %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
+  {% endfor %}
+{% endif %}
+
+{# Print the result #}
+<div class="{{ _css_class }}"{{ _extra_attributes }}>
+  <div class="ecl-container">
+    {%- if breadcrumb is defined -%}
+      {%- include '../ec-component-breadcrumb/breadcrumb.html.twig' with breadcrumb|merge({ extra_classes: 'ecl-page-header__breadcrumb' }) -%}
+    {%- endif -%}
+    {%- if _meta is not empty -%}
+      <div class="ecl-page-header__meta-list">{{ _meta }}</div>
+    {%- endif -%}
+    <h1 class="ecl-page-header__title">{{ _title }}</h1>
+    {%- if _description is not empty -%}
+      <p class="ecl-page-header__description">{{ _description }}</p>
+    {%- endif -%}
+    {%- if infos is defined and infos not empty -%}
```

Comment:
Might worth adding a check for `iterable`, as the React one seems to be ready to accept not only arrays https://github.com/ec-europa/europa-component-library/blob/v2-dev/src/systems/ec/implementations/react/packages/page-header/src/PageHeader.jsx#L17

I am not 100% sure it's necessary though, as in this implementation there is a default of empty array.

Reply:
Fixed in https://github.com/ec-europa/ecl-twig/pull/53/commits/ca4852efa2fa10288e41088c11f32ed3aaeb7ce5